vmdb/app/controllers/application_controller/ci_processing.rb

```diff
@@ -508,7 +508,8 @@ def copy_record
 
     # Build the vm detail gtl view
   def show_details(db, options={})  # Pass in the db, parent vm is in @vm
-    association    = options[:association] || nil
+    association = options[:association] || nil
+    conditions  = options[:conditions] || nil
```

Comment:
Why does this have `|| nil`?  If the value is missing, the hash will return `nil` for that key.